configuration_go/schemas/thanos/common/common.go

```diff
@@ -0,0 +1,40 @@
+package common
+
+import "time"
+
+type LogLevel string
+
+const (
+	LogLevelDebug LogLevel = "debug"
+	LogLevelInfo  LogLevel = "info"
+	LogLevelWarn  LogLevel = "warn"
+	LogLevelError LogLevel = "error"
+)
+
+type LogFormat string
+
+const (
+	LogFormatLogfmt LogFormat = "logfmt"
+	LogFormatJSON   LogFormat = "json"
+)
+
+// Taken from https://github.com/thanos-io/thanos/blob/release-0.32/pkg/model/timeduration.go#L17
+type TimeOrDurationValue struct {
+	Time *time.Time
+	Dur  *time.Duration
+}
+
+// String returns either time or duration.
+func (tdv *TimeOrDurationValue) String() string {
+	switch {
+	case tdv.Time != nil:
+		return tdv.Time.String()
+	case tdv.Dur != nil:
+		if v := *tdv.Dur; v < 0 {
+			return "-" + (-v).String()
+		}
+		return tdv.Dur.String()
+	}
+
+	return "nil"
```

Comment:
This is a bit weird in the Go world. Could we instead return the empty value of the `string` type instead? I think Go developers will be more accustomed to this.

Reply:
I just copy pasted the relevant part for us from this lib. I agree that this does not make sense for our case. They are probably using this do distinguish a zero value of a Time or Duration from a nil. But we're not using this nil. Will remove.

Reply:
Hummm, we can ignore if it'll break something because we're changing this. 🤔

Reply:
Let's leave like that then. It's just copy paste.